Cargo.toml

```diff
@@ -165,11 +165,13 @@ members = [
   "tools/kagami",
   "tools/kura_inspector",
   "tools/parity_scale_decoder",
+  "tools/swarm",
   "version",
   "version/derive",
   "wasm_codec",
   "wasm_codec/derive",
   "wasm_builder",
+  "wasm_builder_cli",
```

Comment:
Can we consider to move `wasm_builder_cli` to tools directory? My point is that `wasm_builder_cli` is a CLI tool.

Reply:
Sounds reasonable. Will do.